src/pages/components/contained-list/style.mdx

```diff
@@ -103,14 +103,19 @@ in a phrase and any proper nouns capitalized.
 
 ## Structure
 
-| Element                     | Property                    | px / rem | Spacing token |
-| --------------------------- | --------------------------- | -------- | ------------- |
-| List header area: on-page   | height                      | 32 / 2   | `$spacing-07` |
-|                             | padding-left, padding-right | 16 / 1   | `$spacing-05` |
-| List header area: disclosed | height                      | 48 / 3   | `$spacing-09` |
-|                             | padding-left, padding-right | 16 / 1   | `$spacing-05` |
-| List item                   | padding-left, padding-right | 16 / 1   | `$spacing-05` |
-| Icon (optional)             | height, width               | 16 / 1   | —             |
+| Element                         | Property                    | px / rem | Spacing token |
+| ------------------------------- | --------------------------- | -------- | ------------- |
+| List header area: on-page       | height                      | 32 / 2   | `$spacing-07` |
+|                                 | padding-left, padding-right | 16 / 1   | `$spacing-05` |
+| List header area: disclosed     | height                      | 48 / 3   | `$spacing-09` |
+|                                 | padding-left, padding-right | 16 / 1   | `$spacing-05` |
+| List item                       | padding-left, padding-right | 16 / 1   | `$spacing-05` |
+| Non-interactive icon (optional) | height, width               | 16px     | —             |
+|                                 | padding left, padding-right | 16 / 1   | `$spacing-05` |
+| Inline action icon (optional)   | height, width               | 16px     | —             |
+|                                 | padding left, padding-right | 16 / 1   | `$spacing-05` |
+| Search icon (optional)          | height, width               | 16px     | —             |
+|                                 | padding left, padding-right | 16 / 1   | `$spacing-05` |
```

Comment:
If we decide to mention the icon button on the Usage tab then we probably need to mention it here.